include/transformations/io-transformations/PrintfToCoutTransformation.h

```diff
@@ -0,0 +1,90 @@
+#ifndef INCLUDE_TRANSFORMATIONS_IO_TRANSFORMATIONS_PRINTFTOCOUTTRANSFORMATION_H_
+#define INCLUDE_TRANSFORMATIONS_IO_TRANSFORMATIONS_PRINTFTOCOUTTRANSFORMATION_H_
+
+#include <yaml-cpp/yaml.h>
+
+#include <string>
+#include <vector>
+#include <memory>
+#include <regex> // NOLINT
```

Comment:
Why do you need regex here? I think all logic with string manipulation encapsulates in another module